dev-client/src/screens/SoilScreen/TextureScreen.tsx

```diff
@@ -0,0 +1,191 @@
+import {Image, ImageSourcePropType} from 'react-native';
+import {selectDepthDependentData} from 'terraso-client-shared/selectors';
+import {
+  updateDepthDependentSoilData,
+  RockFragmentVolume,
+  SoilTexture,
+  textures,
+} from 'terraso-client-shared/soilId/soilIdSlice';
+import {fromEntries, entries} from 'terraso-client-shared/utils';
+import {Icon} from 'terraso-mobile-client/components/Icons';
+import {
+  ImageRadio,
+  radioImage,
+} from 'terraso-mobile-client/components/ImageRadio';
+import {LastModified} from 'terraso-mobile-client/components/LastModified';
+import {FormSelect} from 'terraso-mobile-client/components/form/FormSelect';
+import {FormTooltip} from 'terraso-mobile-client/components/form/FormTooltip';
+import {useNavigation} from 'terraso-mobile-client/navigation/hooks/useNavigation';
+import {
+  SoilPitInputScreenProps,
+  SoilPitInputScreenScaffold,
+} from 'terraso-mobile-client/screens/SoilScreen/components/SoilPitInputScreenScaffold';
+import {useDispatch, useSelector} from 'terraso-mobile-client/store';
+
+const FRAGMENT_IMAGES = {
+  VOLUME_0_1: require('terraso-mobile-client/assets/texture/rock-fragment/1.png'),
+  VOLUME_1_15: require('terraso-mobile-client/assets/texture/rock-fragment/15.png'),
+  VOLUME_15_35: require('terraso-mobile-client/assets/texture/rock-fragment/35.png'),
+  VOLUME_35_60: require('terraso-mobile-client/assets/texture/rock-fragment/60.png'),
+  VOLUME_60: require('terraso-mobile-client/assets/texture/rock-fragment/90.png'),
+} satisfies Record<RockFragmentVolume, ImageSourcePropType> as Record<
+  RockFragmentVolume,
+  ImageSourcePropType
+>;
+
+export const TextureScreen = (props: SoilPitInputScreenProps) => {
+  const {siteId, depthInterval} = props;
+  const {t} = useTranslation();
+  const intervalData = useSelector(selectDepthDependentData(props));
+  const dispatch = useDispatch();
+  const navigation = useNavigation();
+
+  const onTextureChange = useCallback(
+    (texture: SoilTexture) => {
+      dispatch(
+        updateDepthDependentSoilData({
+          siteId,
+          depthInterval: depthInterval.depthInterval,
+          texture,
+        }),
+      );
+    },
+    [dispatch, siteId, depthInterval],
+  );
+
+  const textureOptions = useMemo(
+    () =>
+      fromEntries(
+        textures.map(value => [value, t(`soil.texture.class.${value}`)]),
+      ),
+    [t],
+  );
+
+  const onGuide = useCallback(
+    () => navigation.navigate('TEXTURE_GUIDE', props),
+    [props, navigation],
+  );
+
+  // const onClayChange = useCallback(
+  //   (clay: string) => {
+  //     const parsed = parseInt(clay, 10);
+  //     if (parsed >= 0 && parsed <= 100) {
+  //       dispatch(
+  //         updateDepthDependentSoilData({
+  //           siteId,
+  //           depthInterval: depthInterval.depthInterval,
+  //           // clayPercent: clay,
+  //         }),
+  //       );
+  //     }
+  //   },
+  //   [dispatch, siteId, depthInterval],
+  // );
```

Comment:
issue: please uncomment or remove.

Reply:
this is working code which lillian requested to remove for MVP, i'm leaving commented out so it is easy to add it back again when it's time to implement this feature rather than rewriting or finding an old branch where it exists